drv/i2c-devices/src/lib.rs

```diff
@@ -91,6 +91,36 @@ macro_rules! pmbus_rail_read {
     }};
 }
 
+macro_rules! pmbus_rail_phase_read {
+    ($device:expr, $rail:expr, $phase:expr, $cmd:ident) => {{
+        let rail_payload = [PAGE::CommandData::code(), $rail];
+        let phase_payload = [PHASE::CommandData::code(), $phase];
+
+        match $cmd::CommandData::from_slice(&match $device
+            .write_write_read_reg::<u8, [u8; $cmd::CommandData::len()]>(
+                $cmd::CommandData::code(),
+                &rail_payload,
+                &phase_payload,
+            ) {
+            Ok(rval) => Ok(rval),
+            Err(code) => Err(Error::BadRead {
+                cmd: $cmd::CommandData::code(),
+                code,
+            }),
+        }?) {
+            Some(data) => Ok(data),
+            None => Err(Error::BadData {
+                cmd: $cmd::CommandData::code(),
+            }),
+        }
+    }};
+
+    ($device:expr, $rail:expr, $dev:ident::$cmd:ident) => {{
```

Comment:
This seems like incorrect copy-pasta from `pmbus_rail_read!`

Reply:
Agreed -- thanks!